src/vs/workbench/services/notification/common/notificationService.ts

```diff
@@ -3,24 +3,30 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { INotificationService, INotification, INotificationHandle, Severity, NotificationMessage, INotificationActions, IPromptChoice, IPromptOptions, IStatusMessageOptions } from 'vs/platform/notification/common/notification';
+import { INotificationService, INotification, INotificationHandle, Severity, NotificationMessage, INotificationActions, IPromptChoice, IPromptOptions, IStatusMessageOptions, NoOpNotification } from 'vs/platform/notification/common/notification';
 import { INotificationsModel, NotificationsModel, ChoiceAction } from 'vs/workbench/common/notifications';
 import { Disposable, DisposableStore, IDisposable } from 'vs/base/common/lifecycle';
 import { Event } from 'vs/base/common/event';
 import { registerSingleton } from 'vs/platform/instantiation/common/extensions';
 import { ServiceIdentifier } from 'vs/platform/instantiation/common/instantiation';
-import { IAction } from 'vs/base/common/actions';
+import { IAction, Action } from 'vs/base/common/actions';
+import { IStorageService, StorageScope } from 'vs/platform/storage/common/storage';
+import * as nls from 'vs/nls';
 
 export class NotificationService extends Disposable implements INotificationService {
 
 	_serviceBrand: ServiceIdentifier<INotificationService>;
-
+	private readonly neverShowLabel = nls.localize('neverShowAgain', "Don't Show Again");
```

Comment:
I prefer to inline messages, not have them as variables.

Reply:
It's for the sake of reuse in "prompt" and "notify".

Reply:
Yeah don't worry about that, we are good as long as the key used for NLS is the same.

Reply:
Do you mean to make the key a member or just remove the member variable completely?

Reply:
No simply repeat the same nls.localize call twice, i prefer to read the english label in code where it is being used.